charts/kusion/README.md.gotmpl

```diff
@@ -57,16 +63,16 @@
 First, you need to base64 encode your kubeconfig files.
 
 ```shell
-KUBECONFIG_CONTENT1=$(base64 -w 0 /path/to/your/kubeconfig1)
-KUBECONFIG_CONTENT2=$(base64 -w 0 /path/to/your/kubeconfig2)
+KUBECONFIG_CONTENT1=$(cat /path/to/your/kubeconfig-1 | base64 | tr -d '\n')
+KUBECONFIG_CONTENT2=$(cat /path/to/your/kubeconfig-2 | base64 | tr -d '\n')
 ```
 
 Then you can use the following command to install the latest version of Kusion.
```

Comment:
Similar to the previous comment, the change from 'kubeconfig1' to 'kubeconfig-1' might cause compatibility issues. Verify if this change is necessary.